hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -2306,6 +2312,14 @@ public static byte[] getEcdsaPrivateKeyFromSpec(final HapiSpec spec, final Strin
         return privateKeyByteArray;
     }
 
+    public static PrivateKey getEd25519PrivateKeyFromSpec(final HapiSpec spec, final String privateKeyRef) {
+        var key = spec.registry().getKey(privateKeyRef);
+        final var privateKey = spec.keys()
```

Comment:
I realize I wrote this but could probably just return here without adding a new variable just to return

Reply:
Sure.  But it doesn't hurt.  And sometimes it's nice if you want to set a breakpoint right there before the method returns so you can look at locals.  (Whcih are gone if you let the method return and you just look in the debug window for the returned value.  So I actually prefer to see it more often than not.